security/compliance_operator/compliance-operator-remediation.adoc

```diff
@@ -24,8 +26,6 @@ include::modules/compliance-removing-kubeletconfig.adoc[leveloffset=+1]
 
 include::modules/compliance-inconsistent.adoc[leveloffset=+1]
 
-include::modules/compliance-filtering-failed-results.adoc[leveloffset=+1]
-
 [role="_additional-resources"]
```

Comment:
While you are here, please remove the the plain text from the link.

Per the guidance in [Jupiter Source files readiness Checklist](https://docs.engineering.redhat.com/display/JUP/Jupiter+Source+files+readiness+Checklist), these sections should contain links only. No plain text.